docs/lib/storage/fragments/flutter/getting-started/10_preReq.md

```diff
@@ -1,10 +1,6 @@
-* A Flutter application targeting Flutter SDK >= 1.20 (stable version) with Amplify libraries integrated
+* A Flutter application targeting Flutter SDK >= 2.0 (stable version) with Amplify libraries integrated
```

Comment:
FYI - Some of these changes will conflict with the changes in this PR -> https://github.com/aws-amplify/docs/pull/3359/files#diff-bdbaf78fcdc7aba80a62ad7704a10613164cf02547bfb28833f95ca41183cc63

Reply:
Good catch. Fixed.